test/unit/test_vis.py

```diff
@@ -0,0 +1,314 @@
+import os
+
+import pytest
+
+from deepreg.vis import main, string_to_list
+
+
+def test_string_to_list():
+    string = "abc, 123, def, 456"
+    got = string_to_list(string)
+    expected = ["abc", "123", "def", "456"]
+    assert got == expected
+
+
+def test_gif_slices():
+    """Covered by test_main"""
+    pass
+
+
+def test_tile_slices():
+    """Covered by test_main"""
+    pass
+
+
+def test_gif_warp():
+    """Covered by test_main"""
+    pass
+
+
+def test_gif_tile_slices():
+    """Covered by test_main"""
+    pass
+
+
+class TestMain:
+    def __init__(self):
+        self.img_paths = "./data/test/nifti/unit_test/moving_image.nii.gz"
```

Comment:
sorry, @s-sd here for the constants, you can do directly like

```python
class TestMain:
    img_paths = "./data/test/nifti/unit_test/moving_image.nii.gz"
    ddf_path = "./data/test/nifti/unit_test/ddf.nii.gz"
```

I gonna prepare the docs for testing now... lol

Reply:
@mathpluscode, this way is giving me `NameError: name 'abc' is not defined` when running